index.html

```diff
@@ -27,8 +27,167 @@ <h1 id="name">
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/23.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/24.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/25.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
+      </div>
+    </div>
+  </section>
+
+  <section class="class-container2">
+    <img src="img/26.png" alt="">
+    <div class="description-container">
+      <h2 class="project-name">Project name goes here</h2>
+      <div id="language-container">
+        <div class="languages"> <div class="text">HTML/CSS</div></div>
+        <div class="languages"> <div class="text">Ruby on Rails</div></div>
+        <div class="languages"> <div class="text">JavaScript</div> </div>
+      </div>
+      <div class="link-container">
+        <div id="see-this-project">See this project</div>
+        <div id="arrow"><a href="#">&#x2192</a></div>
```

Comment:
- [ ] `required`
- [ ] please add the required `backgrounds` as the Figma template
- [ ] please see screenshots below

`your template`
![image](https://user-images.githubusercontent.com/66230306/198662082-2e8ba030-00a3-480b-8134-f3ac787ff058.png)

`figma template`
![image](https://user-images.githubusercontent.com/66230306/198664498-1c7cfbef-6db1-4b5a-93fd-fe6e9c00d654.png)

Reply:
I already have done all of this according to figma @mwanawabangona . Please see the screenshot
![aaaa](https://user-images.githubusercontent.com/111677605/198682424-bcb54ecb-c218-49fa-a3df-0e128cb05df0.JPG)